packages/react/src/components/CalendarDayGrid/CalendarDayGrid.styles.ts

```diff
@@ -114,7 +114,7 @@ export const styles = (props: ICalendarDayGridStyleProps): ICalendarDayGridStyle
         selectors: {
           [HighContrastSelector]: {
             color: 'WindowText',
-            backgroundColor: 'Window',
+            backgroundColor: 'transparent',
```

Comment:
Why is this being changed in this PR?

Reply:
That is a really good question. I pulled changes from master and it looks like something got messed up? I'm opening a new PR because the VR tests stopped running